A2/plugins/util.py

```diff
@@ -17,3 +19,23 @@ def ping_command(self, event):
         user_bot_latency = (bot_ping - user_ping).total_seconds() * 1000.0
 
         bot.edit('Latency of you to bot: ~{:.2f}ms'.format(user_bot_latency))
+
+    @Plugin.command('choose', '<options:str...>') 
+    def choose_command(self, event, options):
+        """= choose =
+        Randomly choose between two or more options
+        usage    :: $choose <options...>
+        aliases  :: None
+        category :: Utilities
+        == Arguments
+        options  :: A list of options from which one will be chosen
+        == Examples
+        $choose windows\macOS\linux 'I choose: macOS'
+        $choose boil em\mash em\stick em in a stew 'I choose: boil em'
+        $choose a\ \c 'I choose: c'
```

Comment:
Remove this last example, we don't want to encourage people to do this, we just don't want the bot to break if someone does this.